ts/Core/Axis/Tick.ts

```diff
@@ -848,10 +848,19 @@ class Tick {
         // Create new label if the actual one is moved
         if (!moved && (tick.labelPos || label)) {
             labelPos = tick.labelPos || (label as any).xy;
+            // Use the inverted logic for the yAxis
+            if (!axis.isXAxis) {
+                if (inverted) {
+                    inverted = false;
+                } else {
+                    inverted = true;
+                }
+            }
```

Comment:
I don't think it's a good idea. Perhaps instead of `chart.inverted`, we should use `axis.horiz`?